Makefile

```diff
@@ -31,7 +29,8 @@ BUILD_HOSTNAME := $(shell ./build/get-build-hostname.sh)
 LDFLAGS := "-X github.com/open-policy-agent/opa/version.Version=$(VERSION) \
 	-X github.com/open-policy-agent/opa/version.Vcs=$(BUILD_COMMIT) \
 	-X github.com/open-policy-agent/opa/version.Timestamp=$(BUILD_TIMESTAMP) \
-	-X github.com/open-policy-agent/opa/version.Hostname=$(BUILD_HOSTNAME)"
+	-X github.com/open-policy-agent/opa/version.Hostname=$(BUILD_HOSTNAME) \
+	-s"
```

Comment:
Symbols are typically used for stack traces. There are also DWARF structures for debugging. It looks like you can disable that as well ("-w"). That being said, at this stage in the project, it's probably good to keep both on by default. If size really is a concern, an optional input parameter could be used.